source/core/coverage/ut_coverage.pkb

```diff
@@ -0,0 +1,327 @@
+create or replace package body ut_coverage is
+  /*
+  utPLSQL - Version X.X.X.X
+  Copyright 2016 - 2017 utPLSQL Project
+
+  Licensed under the Apache License, Version 2.0 (the "License"):
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  */
+
+  g_skipped_objects ut_object_names;
+  g_schema_names    ut_varchar2_list;
+  g_include_list    ut_object_names;
+  g_exclude_list    ut_object_names;
+  g_file_mappings   ut_coverage_file_mappings;
+
+
+  /**
+   * Private functions
+   */
+  function to_ut_object_list( a_names ut_varchar2_list) return ut_object_names is
+    l_result ut_object_names;
+  begin
+    if a_names is not null then
+      l_result := ut_object_names();
+      for i in 1 .. a_names.count loop
+        l_result.extend;
+        l_result(l_result.last) := ut_object_name(a_names(i));
+      end loop;
+    end if;
+    return l_result;
+  end;
+
+  /**
+  * Public functions
+  */
+  function default_file_to_obj_type_map return ut_key_value_pairs is
+  begin
+    return ut_key_value_pairs(
+        ut_key_value_pair('fnc', 'FUNCTION'),
+        ut_key_value_pair('prc', 'PROCEDURE'),
+        ut_key_value_pair('trg', 'TRIGGER'),
+        ut_key_value_pair('tpb', 'TYPE BODY'),
+        ut_key_value_pair('pkb', 'PACKAGE BODY'),
```

Comment:
Maybe we should still add pkg extension so we at least somewhat cover such clients gathering coverage only for bodies. They won't change the way they develop.